### PR TITLE
[all hosts] Remove unsupported platforms

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: docs/requirement-sets/common/office-add-in-requirement-sets.md [hunk 1]
```diff
@@ -47,215 +47,215 @@ See [Add-in command requirement sets](add-in-commands-requirement-sets.md).
...
+| See [Open Browser Window API requirement sets](open-browser-window-api-requirement-sets.md). | <ul><li>Office.context.ui.openBrowserWindow</li></ul> |
 
 ---
 
 ### PartialTableBindings
 
 | Minimum Office application support | Methods in set |
 |:-----|:-----|
-| Access Web Apps ||
+| <ul><li>Access Web Apps</li></ul> ||
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/requirement-sets/common/office-add-in-requirement-sets.md:178]: Checking with the Access PM about this.

**Review by Rick-Kirkham — APPROVED** [review]

File: docs/manifest/host.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ When defined in the basic manifest (under [OfficeApp](officeapp.md)), the host t
 Specifies the Host type targeted by this add-in. The value must be one of the following:
 
 - `Document` (Word)
-- `Database` (Access)
+- `Database` (Access (obsolete))
```
Comment (AlexJerabek) [inline docs/manifest/host.md:30]: Can we move this to the end of the list?

**Review by AlexJerabek — APPROVED** [review]

File: docs/manifest/allformfactors.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: AllFormFactors element in the manifest file
 description: Specifies the settings for an add-in for all form factors. 
-ms.date: 02/25/2022
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/manifest/allformfactors.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/manifest/desktopsettings.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: DesktopSettings element in the manifest file
 description: Specifies source location and control settings that apply when your mail add-in is used on a desktop computer.
-ms.date: 04/09/2020
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/manifest/desktopsettings.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/manifest/form.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Form element in the manifest file
 description: UX settings for the forms that your mail add-in will use when running on a particular device (desktop, tablet, or phone).
-ms.date: 06/10/2022
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/manifest/form.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/manifest/host.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Host element in the manifest file
 description: Specifies an individual Office application type where the add-in should activate.
-ms.date: 02/25/2022
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/manifest/host.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/manifest/phonesettings.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 title: PhoneSettings element in the manifest file
 description: The PhoneSettings element specifies the source location and control settings that apply when your mail add-in is used on a phone.
-ms.date: 04/09/2020
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/manifest/phonesettings.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/add-in-commands-requirement-sets.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Add-in commands requirement sets
 description: Overview of Office Add-in commands requirement sets.
-ms.date: 02/15/2024
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/add-in-commands-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/dialog-api-requirement-sets.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Dialog API requirement sets
 description: Learn more about the Dialog API requirement sets.
-ms.date: 09/28/2022
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/dialog-api-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/dialog-origin-requirement-sets.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Dialog Origin requirement sets
 description: Learn more about the Dialog Origin requirement sets.
-ms.date: 09/28/2022
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/dialog-origin-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/identity-api-requirement-sets.md [hunk 11]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Identity API requirement sets
 description: Identity API requirement set information for Office Add-ins.
-ms.date: 07/07/2023
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/identity-api-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/image-coercion-requirement-sets.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Image Coercion requirement sets
 description: Support for Image Coercion requirement sets with Office Add-ins across Excel, OneNote, PowerPoint, and Word.
-ms.date: 09/05/2023
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/image-coercion-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/keyboard-shortcuts-requirement-sets.md [hunk 13]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Keyboard Shortcuts requirement sets
 description: Keyboard Shortcuts requirement set information for Office Add-ins.
-ms.date: 09/28/2022
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/keyboard-shortcuts-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/office-add-in-requirement-sets.md [hunk 14]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office Common API requirement sets
 description: Learn more about the Office Common API requirement sets.
-ms.date: 02/29/2024
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/office-add-in-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/open-browser-window-api-requirement-sets.md [hunk 15]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Open Browser Window requirement sets
 description: Specifies which Office platforms and builds support the openBrowserWindow API.
-ms.date: 09/28/2022
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/open-browser-window-api-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/ribbon-api-requirement-sets.md [hunk 16]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Ribbon API requirement sets
 description: Specifies which Office platforms and builds support the dynamic ribbon APIs.
-ms.date: 02/29/2024
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/ribbon-api-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/common/shared-runtime-requirement-sets.md [hunk 17]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Shared runtime requirement sets
 description: Specifies the platforms and Office applications that support the SharedRuntime APIs.
-ms.date: 12/13/2022
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/common/shared-runtime-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/excel/custom-functions-requirement-sets.md [hunk 18]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Custom Functions requirement sets
 description: Details about the Custom Functions requirement sets for Excel JavaScript API.
-ms.date: 01/10/2024
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/excel/custom-functions-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/excel/excel-api-requirement-sets.md [hunk 19]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Excel JavaScript API requirement sets
 description: Office Add-in requirement set information for Excel builds.
-ms.date: 04/17/2023
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/excel/excel-api-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/outlook/outlook-api-requirement-sets.md [hunk 20]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Outlook JavaScript API requirement sets
 description: Learn more about the Outlook JavaScript API requirement sets.
-ms.date: 02/29/2024
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/outlook-api-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/requirement-sets.md [hunk 21]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office client application and platform availability for Office Add-ins
 description: Supported requirement sets for Excel, OneNote, Outlook, PowerPoint, Project, and Word.
-ms.date: 02/29/2024
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/word/word-api-requirement-sets.md [hunk 22]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Word JavaScript API requirement sets
 description: Office Add-in requirement set information for Word.
-ms.date: 01/25/2024
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/word/word-api-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

File: docs/requirement-sets/powerpoint/powerpoint-api-requirement-sets.md [hunk 23]
```diff
@@ -1,7 +1,7 @@
 ---
 title: PowerPoint JavaScript API requirement sets
 description: Learn more about the PowerPoint JavaScript API requirement sets.
-ms.date: 03/29/2024
+ms.date: 04/09/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/powerpoint/powerpoint-api-requirement-sets.md:4]:
```suggestion
ms.date: 04/15/2024
```

**Pull request merged**